peer-pod-controller/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/confidential-containers/cloud-api-adaptor/peer-pod-controller
 
-go 1.18
+go 1.19
```

Comment:
Afaik, this will also need changes to dependencies like controller-mgr etc as the version of golang is tied to operator-sdk version.

Reply:
@bpradipt, added changes. PTAL.
There is currently no CI check for validating the manifests or related code in peer pod controller. It would be good to add a simple check, wdyt?

Reply:
yeah, it'll be a good addition to the CI.

Reply:
Okay, I'll create a follow up PR shortly.